skymp5-server/cpp/server_guest_lib/script_classes/PapyrusLeveledItem.cpp

```diff
@@ -0,0 +1,40 @@
+#include "PapyrusLeveledItem.h"
+
+#include "LeveledListUtils.h"
+#include "WorldState.h"
+#include "script_objects/EspmGameObject.h"
+
+VarValue PapyrusLeveledItem::GetNthForm(VarValue self,
```

Comment:
there is no docs for this specific method (https://ck.uesp.net/wiki/LeveledItem_Script)

but it looks like it only searches for LVLI in espm and returns nth formId as a Papyrus object

see also
https://en.uesp.net/wiki/Skyrim_Mod:Mod_File_Format/LVLI

Reply:
So  this code does, I think. It doesn't make recursive search, just returns Form from formId stored in LeveledList.

Reply:
it does EvaluateList, not just read data from ESPM